backend/btrixcloud/basecrawls.py

```diff
@@ -456,18 +456,23 @@ async def delete_crawls_all_types(
 
         return {"deleted": True}
 
-    async def get_all_crawl_search_values(self, org: Organization):
+    async def get_all_crawl_search_values(
+        self, org: Organization, type_: Optional[str] = None
+    ):
         """List unique names, first seeds, and descriptions from all captures in org"""
-        names = await self.crawls.distinct("name", {"oid": org.id})
-        descriptions = await self.crawls.distinct("description", {"oid": org.id})
-        crawl_ids = await self.crawls.distinct("_id", {"oid": org.id})
-        cids = await self.crawls.distinct("cid", {"oid": org.id})
+        match_query = {"oid": org.id}
+        if type_:
+            match_query["type"] = type_
+
+        names = await self.crawls.distinct("name", match_query)
+        descriptions = await self.crawls.distinct("description", match_query)
+        crawl_ids = await self.crawls.distinct("_id", match_query)
+        cids = await self.crawls.distinct("cid", match_query)
```

Comment:
```suggestion
        cids = await self.crawls.distinct("cid", match_query) if not type_ or type_ == "crawl" else []
```

Does this make sense, since only applies to crawls?

Reply:
We need the cids for both crawls and "all" (no filter) to be able to grab the first seed URLs to populate in the search